draft-ietf-taps-arch.md

```diff
@@ -376,7 +373,7 @@ The diagram below provides a high-level view of the actions and events during th
 
 * Initiate: The primary action that an application can take to create a Connection to a Remote Endpoint, and prepare any required local or remote state to enable the transmission of Messages. For some protocols, this will initiate a client-to-server style handshake; for other protocols, this will just establish local state. The process of identifying options for connecting, such as resolution of the Remote Endpoint, occurs in response to the Initiate call.
 
-* Listen: The action of marking a Listener as willing to accept incoming Connections. The Listener will then create Connection objects as incoming connections are accepted ({{events}}). Listeners by default register with multiple paths, protocols, and local endpoints, unless constrained by Selection Properties. Connections can be accepted on any of the available paths or endpoints.
+* Listen: The action of marking a Listener as willing to accept incoming Connections. The Listener will then create Connection objects as incoming connections are accepted ({{events}}). Listeners, by default, register with multiple paths, protocols, and local endpoints, unless constrained by Selection Properties. Connections can be accepted on any of the available paths or endpoints.
```

Comment:
I don't think I agree with adding the commas here. Can we revert this change?

Reply:
I was quite sure they are optional and improve readability, but can remove them.